SUMMARY.md

```diff
@@ -1,31 +1,44 @@
 # Table of contents
 
 * [Welcome to Mayastor!](README.md)
+* [Scope](scope.md)
+* [Basic Architecture](basic-architecture.md)
 
 ## Quickstart
 
-* [Scope](quickstart/scope.md)
 * [Prerequisites](quickstart/prerequisites.md)
 * [Preparing the Cluster](quickstart/preparing-the-cluster.md)
 * [Deploy Mayastor](quickstart/deploy-mayastor.md)
 * [Configure Mayastor](quickstart/configure-mayastor.md)
+  * [Storage Class Parameters](reference/storage-class-parameters.md)
```

Comment:
Indentation check

Reply:
This was done intentionally, wanted to move 'Storage class' section within 'Configure Parameter'.